src/functional/ffront/decorator.py

```diff
@@ -119,6 +129,33 @@ def from_function(
             definition=definition,
         )
 
+    def _lowered_funcs_from_closureref(
+        self, captured_vars: CapturedVars
+    ) -> list[itir.FunctionDefinition]:
+        lowered_funcs = []
+
+        vars_ = collections.ChainMap(captured_vars.globals, captured_vars.nonlocals)
+        for name, value in vars_.items():
+            # With respect to the frontend offsets are singleton types, i.e.
+            #  they do not store any runtime information, but only type
+            #  information. As such we do not need their value.
+            if isinstance(value, Offset):
```

Comment:
Is `functional.iterator.runtime.Offset` the right type here? Shouldn't it be `fbuiltins.FieldOffset`?

(Check: https://github.com/GridTools/gt4py/pull/640/files#diff-5a2d8f3beae7064a7845cfddc917538e55d53d9d3694a978d2e8b2c5a72ab3ae)